modules/gatsby/src/components/styled/index.js

```diff
@@ -81,7 +81,7 @@ export const BodyGrid = styled('div', ({$theme, ...props}) => ({
   display: 'grid',
   gridTemplateRows: '64px 1fr',
   gridTemplateColumns: '300px 1fr',
-  maxWidth: `${$theme.breakpoints.large}px`,
+  // maxWidth: `${$theme.breakpoints.large}px`,
```

Comment:
we can just delete it. we don't want it back

Reply:
deleted